src/ui/wxWidgets/TreeCtrl.cpp

```diff
@@ -703,6 +718,8 @@ void TreeCtrl::AddItem(const CItemData &item)
   wxTreeItemId gnode = AddGroup(GroupNameOfItem(item));
   const wxString disp = ItemDisplayString(item);
   wxTreeItemId titem = AppendItem(gnode, disp, -1, -1, data);
+  if(GetRootItem() != gnode && GetItemImage(gnode) == EMPTY_NODE_II)
+    wxTreeCtrl::SetItemImage(gnode, NODE_II); // Could be empty before
```

Comment:
Seems to have the same purpose as the code on line 519 and 520. How about a little helper function to avoid code duplication?

Reply:
Demanded change is handled in "Use Function to set EMPTY_NODE_II or set back to NODE_II."

Reply:
This change is already in master repository. I don‘t know what is wrong with my local repository. The last changes / commit are repeated... and k do not know how to solve this problem except setting back the repository to master. But next time the story is repeated.